pkgs/test_api/lib/src/frontend/expect.dart

```diff
@@ -138,8 +138,9 @@ Future _expect(actual, matcher,
 
   var matchState = {};
   try {
-    if ((matcher as Matcher).matches(actual, matchState))
+    if ((matcher as Matcher).matches(actual, matchState)) {
```

Comment:
Did we add a lint for this recently? Not opposed to the clean up but it makes the PR hard to follow. Feel free to open an issue and I'll just fix all packages in a single go. I'd drop these type of changes from this PR.

Reply:
It's true that it's unrelated.
Reverted these changes from here, and filed http://go/gh/dart-lang/test/issues/1032.
Once that is fixed I'll merge and make sure that Travis succeeds on this PR.

Reply:
I see. I didn't realize we now make use of `package:pedantic` for lints. Changes to that package will break us. Fix is here: https://github.com/dart-lang/test/pull/1033

Sorry about that.

Reply:
The fix is in (thanks!), merged now.